postgresql-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+org = "ballerinax"
+name = "postgresql"
+version = "@toml.version@"
+
+[[platform.java11.dependency]]
+path = "../postgresql-native/build/libs/postgresql-native-0.1.0.jar"
+groupId = "org.ballerinalang"
+artifactId = "postgresql-native"
+version = "@project.version@"
+
+[[platform.java11.dependency]]
+path = "@project.path@/repo/bala/ballerina/sql/@stdlib.sql.version@/java11/platform/java11/sql-native-@stdlib.sql.native.version@.jar"
+groupId = "org.ballerinalang"
+artifactId = "ballerina-sql"
+version = "@stdlib.sql.native.version@"
+
+[[platform.java11.dependency]]
+path = "./lib/postgresql-42.2.18.jar"
+artafactId = "postgresql-connector-java"
+groupId = "postgresql"
+scope = "testOnly"
+version = "42.2.18."
```

Comment:
```suggestion
version = "42.2.18"
```

Reply:
I will change

Reply:
Updated
Commit id is 470c221da997cced9e1bdb6676e8d02f8ffc5c2b